src/components/Brushbox.tsx

```diff
@@ -1,15 +1,27 @@
 import { useState } from "react"
 import './Brushbox.css';
 import { Icon, IconMap } from '../Model/icon';
+import { BrushSelection } from "../context";
+
+interface BrushBoxProps {
+  setBrushSelection: (selection: BrushSelection) => void;
+}
 
 
-const BrushBox = ({setBrushSelection}) => {
-  const IconList: any[] = [];
-  const [clicked, setClicked] = useState(-1)
+const BrushBox = (props: BrushBoxProps) => {
+  const {setBrushSelection} = props
+  const iconList: JSX.Element[] = [];
+  const [clicked, setClicked] = useState(-2)
```

Comment:
this also doesn't seem necessary anymore. Let's move towards this component ideally being as stateless as possible and get its state from props instead.